tritium-metrics-jvm/src/test/java/com/palantir/tritium/metrics/jvm/JvmMetricsTest.java

```diff
@@ -71,7 +72,7 @@
             "jvm.memory.non-heap.max",
             "jvm.memory.heap.max",
             "jvm.safepoint.time",
-            "jvm.threads.timed_waiting.count",
+            "jvm.threads.timed-waiting.count",
```

Comment:
This caused tests to fail locally, I'm not sure how it worked on develop.

Reply:
that's quiet odd